### DISCUSSION TITLE
Staying in edit mode across cells?

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1716] [Q&A] [answered]
If I want to edit a bunch of cells (e.g., do some data entry), I find having to hit `e` on every cell hard -- goes against my spreadsheet muscle memory. I know vd isn't really setup for entry as much as analysis, but wonder if/how others manage?

**Comment by saulpw** [comment, marked as answer]
Shift+Arrow will move and keep the edit mode active.

**Reply by reagle** [reply]
Thank you. I now see that is in the manpage, but I'm more likely to check the [cheatsheet](https://jsvine.github.io/visidata-cheat-sheet/en/). I'm not sure who owns the [cheatsheet](https://jsvine.github.io/visidata-cheat-sheet/en/) but could this be added there? (Or could there be an official cheat sheet in this repo? Or is that what the manpage is an the cheatsheet is redundant?)

**Reply by saulpw** [reply]
@jsvine created and owns that cheatsheet; you might suggest this to him.  The manpage is our internal "cheatsheet" that should have (almost) all of the commands searchable by some reasonable keyword.  So feel free to print out his cheatsheet for the most commonly commands, but then be prepared to go to the manpage (with Ctrl+H) and search there for others.

**Reply by daviewales** [reply]
Should this also be in the command list? Or is it not technically a command?

**Reply by saulpw** [reply]
Yeah, not technically a command. Maybe there should be multiple cheatsheets, and this could be on the one for "Adding and Editing Data".   How much would it help to make a `cheatsheets/` folder in the visidata repo, where we could add simple markdown files with these commands and some quick hints about how to use them?  I'm open to lots of options.

**Reply by reagle** [reply]
I think I'd prefer a single (with sections) to multiple sheets, but it would be nice if it was easily accessible for contributions, including mnemonics behind some of the key sequences.

**Reply by saulpw** [reply]
Yeah, it would be great if it were more accessible for contributions.  I'm thinking that one file per section might be helpful in that regard, and if they're just plain markdown, then they could be concatenated together to make the single cheatsheet you want.  It would be easier to start the "editing cheatsheet" if there wasn't the burden of having to make or port the whole toplevel cheatsheet first.
